tests/openvino/test_export.py

```diff
@@ -76,12 +76,17 @@ class ExportModelTest(unittest.TestCase):
         "stable-diffusion-xl": OVStableDiffusionXLPipeline,
         "stable-diffusion-xl-refiner": OVStableDiffusionXLImg2ImgPipeline,
         "latent-consistency": OVLatentConsistencyModelPipeline,
-        "llava": OVModelForVisualCausalLM,
```

Comment:
why do you remove this test?

Reply:
it's not removed but added only when transformers >= v4.37.2 as for LlavaOpenVINOConfig `MIN_TRANSFORMERS_VERSION` is set to v4.37.2 (see below in test)

https://github.com/huggingface/optimum-intel/blob/3acf13f062f1e987268218232d474bdae0e9f2aa/optimum/exporters/openvino/model_configs.py#L1805